cms/djangoapps/contentstore/toggles.py

```diff
@@ -219,6 +219,24 @@ def use_new_custom_pages(course_key):
     """
     return ENABLE_NEW_STUDIO_CUSTOM_PAGES.is_enabled(course_key)
 
+# .. toggle_name: contentstore.new_studio_mfe.use_markdown_editor
```

Comment:
We are in the process of retiring the `new_studio_mfe` namespace as part of [the legacy studio frontend DEPR](https://github.com/openedx/edx-platform/issues/36275). Could you name this `contentstore.react_markdown_editor` instead?